tools/psa/__init__.py

```diff
@@ -0,0 +1,17 @@
+from os.path import basename
+
+
+# Find secure image.
```

Comment:
Could you change this into a doc comment? You would place this in a string as the first statement inside the body of the function.

E.G.

```python
def find_secure_image(toolchain, resources, ns_image_path, configured_s_image_path, image_type):
    """ Find secure image. """
```

Reply:
done